### PR TITLE
MGMT-12039: Add e2e-metal-assisted-lvm jobs

### PR DESCRIPTION
**Pull request by adriengentil**

### CODE REVIEW

**Review by osherdp — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-cloud_hotfix_releases-presubmits.yaml [hunk 1]
```diff
@@ -519,6 +519,81 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-assisted-ipv4v6,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
[the remark this answers is not in the document]
Comment (adriengentil) [reply ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-cloud_hotfix_releases-presubmits.yaml:523]: :man_facepalming: ...but I thought about the `reporter_config`!

**Review by osherdp — APPROVED** [review]

**Comment by adriengentil** [comment]
```ERROR    root:waiting.py:255 Hosts statuses are: [(1, '93e81e55-b9ac-4750-b223-5b6f6318ca4e', 'test-infra-cluster-c8d90384-worker-1', 'worker', 'insufficient', 'Host cannot be installed due to following failing validation(s): ODF LVM requires at least one non-installation HDD/SSD disk on the host (minimum size: 0 GB) ; Require at least 3 CPU cores for worker role, found only 2'), (2, 'a50aca45-2370-43d2-9cd8-c7845991be42', 'test-infra-cluster-c8d90384-master-0', 'master', 'insufficient', 'Host cannot be installed due to following failing validation(s): Require at least 5 CPU cores for master role, found only 4'), (3, '39debd96-cbf4-43e8-b45e-f0c024cf1e7f', 'test-infra-cluster-c8d90384-master-1', 'master', 'insufficient', 'Host cannot be installed due to following failing validation(s): Require at least 5 CPU cores for master role, found only 4'), (4, '059db73c-4c08-46ba-acfb-d70d068472ae', 'test-infra-cluster-c8d90384-worker-0', 'worker', 'insufficient', 'Host cannot be installed due to following failing validation(s): ODF LVM requires at least one non-installation HDD/SSD disk on the host (minimum size: 0 GB) ; Require at least 3 CPU cores for worker role, found only 2'), (5, '6a007a2d-b76d-468f-bad4-8563b54819cf', 'test-infra-cluster-c8d90384-master-2', 'master', 'insufficient', 'Host cannot be installed due to following failing validation(s): Require at least 5 CPU cores for master role, found only 4')]
```
Looks like lvm as specific requirements

File: ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml [hunk 2]
```diff
@@ -2453,6 +2453,81 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )edge-e2e-metal-assisted-kube-api-net-suite,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (osherdp) [inline ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml:2457]: Remember to change this

**Review by osherdp — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml [hunk 3]
```diff
@@ -845,6 +845,17 @@ tests:
...
+- as: e2e-metal-assisted-lvm
+  optional: true
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        OLM_OPERATORS=lvm
+        MASTERS_COUNT=1
+        MASTER_MEMORY=18000
+        MASTER_CPU=9
```
Comment (adriengentil) [inline ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:857]: the configuration for lvm in test-infra is righty defined here: https://github.com/openshift/assisted-test-infra/blob/master/src/consts/olm_operators.py#L83-L87
but is actually broken by the logic that tries to merge(?) user defined values, trigger defined values and olm defined values: https://github.com/openshift/assisted-test-infra/blob/master/src/assisted_test_infra/test_infra/utils/operators_utils.py#L100-L101

I propose to set the values here in the jobs (if it works), and to follow up with a fix for test-infra later.

**Review by osherdp — APPROVED** [review]

**Review by osherdp — APPROVED** [review]

**Pull request merged**